.github/workflows/trunk.yaml

```diff
@@ -0,0 +1,66 @@
+name: Version Bump
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ set-up-actions ]
+    types: [closed]
+
+jobs:
+  get_bump_level:
+    if: ${{ github.repository == 'wwt/WorkflowSchema' && github.event.pull_request.merged == true }}
+    runs-on: ubuntu-latest
+    outputs:
+      bump_level: ${{ steps.step1.outputs.bump_level }}
+    steps:
+      - id: step1
+        run: |
+          BUMP_LEVEL=$(gh pr \
```

Comment:
So `gh` is just available on the ubuntu-latest image?

Reply:
[yes it is](https://docs.github.com/en/actions/advanced-guides/using-github-cli-in-workflows)